tiledb/sm/query/writers/writer_base.cc

```diff
@@ -924,8 +924,9 @@ Status WriterBase::filter_tile(
       &filters, array_->get_encryption_key()));
 
   // Check if chunk or tile level filtering/unfiltering is appropriate
-  bool use_chunking = filters.use_tile_chunking(
-      array_schema_.var_size(name), array_schema_.version(), tile->type());
+  // bool use_chunking = filters.use_tile_chunking(
```

Comment:
Reset?

Reply:
Fixed + max_chunk_size -> no chunking in https://github.com/TileDB-Inc/TileDB/pull/3969/commits/553ec8762d76ac719a3f85a9b8de828fb659874c